mrt-services/ldap/start_import.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+# start openDJ (detached) 
```

Comment:
The solr container that is in place for Dryad needs to perform a similar function that starts the service and then populates the content.